doc/taoQtiTestPreviewer.yaml

```
@@ -0,0 +1,249 @@
+openapi: 3.0.1
+info:
+  title: taoTestPreview API
+  description: Stands for TAO test Previewer API.
+  version: 1.0.0
+servers:
+  - url: '{server}'
+    variables:
+      server:
+        default: https://community.docker.localhost/
+components:
+  securitySchemes:
+    cookieAuth:
+      type: apiKey
+      in: cookie
+      name: tao_community
+
+  responses:
+    init:
+      description: Good response
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/testInit'
+
+    badResponse:
```

```suggestion
    BadResponse:
```

And do the same for the others.